src/compiler/emitter.ts

```diff
@@ -1613,6 +1613,10 @@ namespace ts {
                         return emitJSDocSignature(node as JSDocSignature);
                     case SyntaxKind.JSDocTag:
                     case SyntaxKind.JSDocClassTag:
+                    case SyntaxKind.JSDocPublicTag:
+                    case SyntaxKind.JSDocPrivateTag:
+                    case SyntaxKind.JSDocProtectedTag:
+                    case SyntaxKind.JSDocOverrideTag:

```

Comment:
can you add an emit test of this? Preferably including `// @declaration: true`

You'll need `// @outdir: out/` and `// @allowjs: true` too.

Reply:
sure, so what about I revert such changes if there are no good reason to change this part ? @Kingwl